### ISSUE TITLE
Problem: RustFBP is on nightly

### ISSUE THREAD

**Issue by dmichiels** [fractalide/fractalide#275]
Solution: seems that the global allocator is now a stable feature : https://blog.rust-lang.org/2018/08/02/Rust-1.28.html

We can theoretically move to stable.

**Comment by sjmackenzie** [comment]
Yes I noticed! That'll be a good step forward. I hope we can move onto the rust side of things as soon as we get something minimally working with the rkt hyperflow.

**Comment by sjmackenzie** [comment]
We'll need to sort out https://github.com/fractalide/fractalide/issues/277 before we can start in anger on the rust stuff.

**Comment by clacke** [comment]
At least nightly is pinned now, since #229 , so nothing will move under our feet.
